src/main/java/me/ramidzkh/mekae2/ae2/MekanismKey.java

```diff
@@ -111,21 +112,21 @@ protected Component computeDisplayName() {
     @Override
     public void addDrops(long amount, List<ItemStack> drops, Level level, BlockPos pos) {
         if (stack instanceof GasStack gasStack) {
-            MekanismAPI.getRadiationManager().dumpRadiation(new Coord4D(pos, level),
+            IRadiationManager.INSTANCE.dumpRadiation(new Coord4D(pos, level),
                     ChemicalBridge.withAmount(gasStack, amount));
         }
     }
 
     @Override
     public boolean isTagged(TagKey<?> tag) {
         if (stack.getType() instanceof Gas gas) {
-            return tag.registry().equals(MekanismAPI.gasRegistryName()) && gas.is((TagKey<Gas>) tag);
+            return tag.registry().equals(MekanismAPI.gasRegistry()) && gas.is((TagKey<Gas>) tag);
         } else if (stack.getType() instanceof InfuseType infuse) {
-            return tag.registry().equals(MekanismAPI.infuseTypeRegistryName()) && infuse.is((TagKey<InfuseType>) tag);
+            return tag.registry().equals(MekanismAPI.infuseTypeRegistry()) && infuse.is((TagKey<InfuseType>) tag);
         } else if (stack.getType() instanceof Pigment pigment) {
-            return tag.registry().equals(MekanismAPI.pigmentRegistryName()) && pigment.is((TagKey<Pigment>) tag);
+            return tag.registry().equals(MekanismAPI.pigmentRegistry()) && pigment.is((TagKey<Pigment>) tag);
         } else if (stack.getType() instanceof Slurry slurry) {
-            return tag.registry().equals(MekanismAPI.slurryRegistryName()) && slurry.is((TagKey<Slurry>) tag);
+            return tag.registry().equals(MekanismAPI.slurryRegistry()) && slurry.is((TagKey<Slurry>) tag);
```

Comment:
These should be `MekanismAPI.GAS_REGISTRY_NAME` etc as you are comparing a `ResourceKey` to an `IForgeRegistry`